src/main/java/world/inetum/realdolmen/inetumrealJobs/entities/User.java

```diff
@@ -46,177 +51,31 @@
     @Column(name = "box")
     private String box;
     @NotBlank
+    @Pattern(regexp = "^[A-Za-z0-9ñÑáéíóúÁÉÍÓÚ ]+$")
     @Column(name = "city")
     private String city;
     @NotBlank
+    @Pattern(regexp = "^[0-9]*$")
     @Column(name = "postal_code")
     private String postalCode;
     @Column(name = "country")
+    @Pattern(regexp = "^[A-Za-z0-9ñÑáéíóúÁÉÍÓÚ]+$")
     @NotBlank
     private String country;
+    @Size(min = 12, max = 12)
```

Comment:
Careful here, some people use country codes, some don't and not all phone numbers have the same length